app/components/primer/beta/breadcrumbs.pcss

```diff
@@ -1,7 +1,6 @@
 .breadcrumb-item {
   display: inline-block;
   margin-left: -0.35em;
-  white-space: nowrap;
```

Comment:
If I recall correctly, I think this was put in because of the truncation that's sometimes used with the breadcrumbs. Could you write a lookbook preview with the `truncate-css` and `Truncate` component to see if there's any bad effects there?

Reply:
Sure! To clarify, are you asking for two stories—one using [primer/css’ truncate](https://github.com/primer/css/tree/main/src/truncate) and one using [primer/view_components’ Truncate](https://primer.style/components/truncate/rails/beta#examples)? Or, does the Truncate component use the truncate CSS under-the-hood, such that only a single story is necessary?

Reply:
I think you could use the deprecated [Primer::Truncate](https://github.com/primer/view_components/blob/main/app/components/primer/truncate.rb) for the CSS one and the [Primer::Beta::Truncate](https://github.com/primer/view_components/blob/main/app/components/primer/beta/truncate.rb) for the standard way.

Reply:
Stories added!
- [WithDeprecatedTruncate](http://primer-view-components-preview-2869.eastus.azurecontainer.io/view-components/lookbook/inspect/primer/beta/breadcrumbs/with_deprecated_truncate)
- [WithBetaTruncate](http://primer-view-components-preview-2869.eastus.azurecontainer.io/view-components/lookbook/inspect/primer/beta/breadcrumbs/with_beta_truncate)

I don’t see any bad effects from removing `white-space: nowrap`.